src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -97,8 +97,17 @@ class ExplorerIndividualContent extends Component {
             ? matchingMenuKeys(individualMenu, urlPath(this.props.nodeInfo.CHORD_URL))
             : [];
 
+        const headerTitle = (individual) => {
+            if (!individual) {
+                return null;
+            }
+            const mainId = (individual || {}).id;
```

Comment:
no longer need `|| {}` since you check falsey-ness above

Reply:
thanks.... added `if (!individual)` at the last moment and clearly didn't bother re-reading the rest of the code